src/components/DataTable/DataTable.stories.js

```diff
@@ -104,6 +108,13 @@ export default {
       name: 'items',
       description: description.items,
     },
+    keepSelectedOnChange: {
+      name: 'isLoading',
+      description: description.keepSelectedOnChange,
+      control: {
+        type: 'boolean',
+      },
+    },
```

Comment:
Hi @gustavomelki nice Job, but at this point the action name is wrong, i believe it should be "keepSelectedOnChange", right?

Reply:
@fernandacanto I think it's already with this name, no?

Reply:
@fariacc No, on line 112 we have the attribute "name" which is like "isLoading".

Reply:
With that in Control on blok.ink it presents two isLoading options, as in the image below.
![Captura de Tela 2022-03-22 às 14 27 22](https://user-images.githubusercontent.com/8209305/159540356-35e08a80-3a2a-403e-9e49-7d9b63faf11a.png)